.sphinx/_static/custom.css

```diff
@@ -193,3 +193,26 @@ details summary {
 .highlight button.copybtn {
    opacity: 0;
 }
+
+/* Mimicking the 'Give feedback' button for UX consistency */
+.sidebar-search-container input[type=submit] {
+    border: 2px solid #d6410d;
+    padding: var(--sidebar-search-input-spacing-vertical) var(--sidebar-search-input-spacing-horizontal);
+    background: #d6410d;
+    font-weight: bold;
+    font-size: var(--font-size--small);
+    cursor: pointer;
+}
+
+.sidebar-search-container input[type=submit]:hover {
+    text-decoration: underline;
+}
+
+.highlight .o {
+    color: #bb5400;
+}
+
+.highlight .s,
+.highlight .s2 {
+    color: #3f8100;
+}
```

Comment:
Do you remember any examples for that? I can't find any ...
Did you test the colours for both the light and the dark theme?

Reply:
I think it's fine if the checks run on the light theme only - that means we provide an accessible option.
But when updating the style sheet, we should manually (or visually I guess :wink: ) inspect both variants.